add FEM example to documentation

jorgepz opened the pull request:
## PR description
A FEM example page is added to the documentation.

lucaferranti commented:
Fantastic, thanks a lot!

A couple of comments

1. I think this would better fit under "applications" folder?
2. To actually tell Documenter.jl to include the page in the documentation, you should also update the [website structure](https://github.com/JuliaIntervals/IntervalLinearAlgebra.jl/blob/main/docs/make.jl#L24) in `docs/make.jl`. In this case the line linked could be changed to `applications => ["Title " => "path/to/file"],`. Or was it intentional to leave it out for now?

(Note to myself, I should add a note about how to add new pages to the docs in the contribution guidelines)

lucaferranti commented:
Writing all matrices in markdown sounds indeed painful. I like the idea of using Literate.jl. I think there are two options

1) we update the `docs/make.jl` file to call Literate and automatically generate the markdown files from it when deploying,

2) we manually generate the markdown locally with Literate before pushing.

The second one is kind of quick and dirty and maybe ok for just one tutorial, the first one is I guess more robust and professional.

Which one do you prefer for this PR? I'd be glad to setup the Literate+Documenter workflow for you if you want

lucaferranti commented:
Ah apparently, the preview feature is available only if the PR is opened from a branch in the same repository, sorry for that :/

If you want to deploy it locally to see what it looks like, in principle navigating into the `docs` folder and calling
```
julia --project make.jl
```

should be enough. Then by opening `docs/build/index.html` you can check what it looks like. If the previous step failed, then it's likely I have some weird boo-boo in the docs that for some reason went undetected so far.

Anyway, I was able to clone the PR and deploy that locally, so I can check that everything looks fine

lucaferranti commented:
I fixed a few latex/literate issues. Regarding the figure, when I deployed locally the diagram itself was relatively small and it had a huge white background around it. How was the figure created? I think iin some softwares sometimes you have to manually "crop" the image to avoid a huge white background around it

jorgepz commented:
> I fixed a few latex/literate issues. Regarding the figure, when I deployed locally the diagram itself was relatively small and it had a huge white background around it. How was the figure created? I think iin some softwares sometimes you have to manually "crop" the image to avoid a huge white background around it

thanks for the comment. I am using Inkscape. I generated the png and moved it to the assets folder.
